internal/admission/defaulting_test.go

```diff
@@ -220,52 +220,60 @@ func TestFlow(t *testing.T) {
 			//THEN
 			require.NotNil(t, res)
 			require.True(t, res.AdmissionResponse.Allowed)
-			require.Equal(t, tt.want.patchesCount, len(res.Patches))
-			if tt.want.patchesCount > 0 {
-				patchValue := (res.Patches[0].Value).(map[string]interface{})
-				require.Contains(t, patchValue, pkg.PodValidationLabel)
-				require.Equal(t, tt.want.validationStatus, patchValue[pkg.PodValidationLabel])
+
+			if tt.shouldCallValidate {
+				mockImageValidator.AssertNumberOfCalls(t, "Validate", 1)
```

Comment:
Can we replace it with `expectedPatch`?

Reply:
done

Reply:
Different response patches are produced by those 2 functions even when nothing to do
- admission.Allowed("validation is not needed for pod")
- admission.PatchResponseFromRaw(req.Object.Raw, fBytes)